react-native-sdk/index.tsx

```diff
@@ -22,6 +22,7 @@ interface IAppProps {
     meetingOptions: {
         domain: string;
         roomName: string;
+        jwt: string;
```

Comment:
There are more options, aren't we missing any?

Reply:
Are you referring to #13630 ones?